megamek/src/megamek/client/ui/swing/lobby/PlayerSettingsDialog.java

```diff
@@ -407,8 +404,9 @@ private void setupValues() {
         txtWidth.setText(Integer.toString(player.getStartWidth()));
         txtOffset.setText(Integer.toString(player.getStartOffset()));
 
-        int bh = clientgui.getClient().getMapSettings().getBoardHeight();
-        int bw = clientgui.getClient().getMapSettings().getBoardWidth();
+        MapSettings ms = clientgui.getClient().getMapSettings();
+        int bh = ms.getBoardHeight() * ms.getMapHeight();
```

Comment:
Is it... possible that the maps are not all the same size?

Reply:
when you switch size, it switches the size for all individual maps.